tests/test_annotations.rs

```diff
@@ -20,6 +20,13 @@ struct Rename {
     a2: i32,
 }
 
+#[derive(Debug, PartialEq, Serialize, Deserialize)]
+struct DirectionRename {
+    a1: i32,
+    #[serde(rename_ser="a3", rename_de="a4")]
```

Comment:
Overall this looks good, but I think this attribute should probably be `rename_serialize` and `rename_deserialize`. That'll be more familiar with folks that aren't used to the short names (which should probably also be renamed).